src/simulation/ApplyLoad.h

```diff
@@ -0,0 +1,33 @@
+#include "main/Application.h"
+#include "simulation/TxGenerator.h"
+#include "test/TestAccount.h"
+
+namespace stellar
+{
+class ApplyLoad : public TxGenerator
```

Comment:
nit: Does this really need to inherit from `TxGenerator`? You don't seem to overload anything and I'm not sure how much sense does this make semantically.

Reply:
Yeah this was just simpler to do during the refactor, but served no purpose other than that. I updated `TxGenerator` to just be a member.